Add note about SET SESSION privilege limitation (#8652)
crate/crate: blackbox/docs/admin/privileges.rst

and ``SET SESSION`` statements, as well as using the available user defined
functions, on the object for which the privilege applies.

.. NOTE::

   :ref:`SET GLOBAL <ref-set-desc>` privileges cannot be granted because ``SET

>    GLOBAL`` statements can only be issued by the superuser.```

Reviewer: ```suggestion
   GLOBAL`` statements can only be issued by the superuser.
```

and ``SET SESSION`` statements, as well as using the available user defined
functions, on the object for which the privilege applies.

.. NOTE::

>    DQL privileges only be granted for ``SET SESSION`` statements.

Reviewer: oops. there's a typo here. missing a "can" before "only"

Reviewer 2: Doesn't this just repeat what's already written in the sentence above?

>    DQL privileges only be granted for ``SET SESSION`` statements.

Reviewer: ```suggestion

```

> 

Reviewer: ```suggestion
```

>    :ref:`SET GLOBAL <ref-set-desc>` privileges cannot be granted to regular

Reviewer: ```suggestion
   :ref:`SET GLOBAL <ref-set-desc>` privileges cannot be granted because ``SET
   GLOBAL`` statements can only be issued by the superuser.```

>    users. ``SET GLOBAL`` statements can only be issued by the superuser.

Reviewer: ```suggestion
```
